analysis_driver/quality_control/calculate_relatedness.py

```diff
@@ -1,41 +1,50 @@
-from egcg_core import executor, util
+import os
+from egcg_core import executor, util, clarity
 from luigi import Parameter, ListParameter
 from analysis_driver.config import default as cfg
 from analysis_driver.segmentation import Stage
 from analysis_driver.util.bash_commands import java_command
 
-
-class Relatedness(Stage):
-    gvcf_files = ListParameter()
+class Genotype_gVCFs(Stage):
+    gVCFs = ListParameter()
     reference = Parameter()
-    project_id = Parameter()
 
     @property
     def gatk_outfile(self):
-        return self.project_id + '_genotype_gvcfs.vcf'
+        return os.path.join(self.job_dir, self.dataset.name + '_genotype_gvcfs.vcf')
 
     def gatk_genotype_gvcfs_cmd(self):
-        gvcf_variants = ' '. join(['--variant ' + util.find_file(i) for i in self.gvcf_files])
+        gvcf_variants = ' '. join(['--variant ' + util.find_file(i) for i in self.gVCFs])
         number_threads = 12
         return java_command(memory=20, tmp_dir=self.job_dir, jar=cfg['tools']['gatk']) + \
                ' -T GenotypeGVCFs -nt %s -R %s %s -o %s' % (
             number_threads, self.reference, gvcf_variants, self.gatk_outfile
         )
 
-    def vcftools_relatedness_cmd(self):
-        return '%s --relatedness2 --vcf %s --out %s' % (
-            cfg['tools']['vcftools'], self.gatk_outfile, self.project_id
-        )
-
     def run_gatk(self):
         return executor.execute(
             self.gatk_genotype_gvcfs_cmd(),
             job_name='gatk_genotype_gvcfs',
             working_dir=self.job_dir,
-            cpus=12,
-            mem=30
+            cpus=30,
+            mem=50
```

Comment:
Is this really necessary? 
Especially since the command you're running only has 12 threads?